crates/solver/src/arguments.rs

```diff
@@ -333,6 +333,11 @@ pub struct Arguments {
 
     #[clap(flatten)]
     pub score_params: score_computation::Arguments,
+
+    /// Should we skip settlements with non-positive score for solver
+    /// competition?
+    #[clap(long, env, default_value = "true")]
```

Comment:
Using `default_value = true` on its own means we can't disable the value. However, Valentin found out that this can be addressed by adding `action = clap::ArgAction::Set` which changes the syntax from `--bool-flag` to `--bool-flag <true|false>`. This should be used from now on for every new bool argument.

Reply:
Nice catch. Did not check similar bool arguments. Fixed.